git_theta/git_utils.py

```diff
@@ -174,11 +174,13 @@ def add_filter_theta_to_gitattributes(gitattributes: List[str], path: str) -> st
                 pattern_found = True
                 if not "filter=theta" in match.group("attributes"):
                     line = f"{line.rstrip()} filter=theta"
+                if not "merge=theta" in match.group("attributes"):
```

Comment:
Can we change this function name to `add_theta_to_gitattributes` since it's doing filters, merge drivers, and eventually diff drivers.

Reply:
Done